### ISSUE TITLE
Option to always enable polyfill by domain

### ISSUE THREAD

**Issue by AaronSchulz** [micolous/gpsd-chrome-polyfill#1]
It would be nice if there was an option to to always use (or not use) the polyfill once the satellite icon is clicked on websites. That way google maps and open street maps, for example, could always use the polyfill.

**Comment by micolous** [comment]
Agreed, but the better way is for [Chrome to actually provide an API for location providers](https://bugs.chromium.org/p/chromium/issues/detail?id=594389).

It's probably possible with [optional permissions](https://developer.chrome.com/apps/permissions),  but then you're extending what is essentially a hack around Chrome's limited behaviour.

I'd happily take a PR for this -- I'm unlikely to work on this any time soon.

**Comment by sunshuvo** [comment]
any update on this.

**Comment by micolous** [comment]
I haven't developed this extension since 2017, and I don't use Chromium-based browsers anymore. :)

If you don't like this situation, please [complain to the Chromium developers](https://bugs.chromium.org/p/chromium/issues/detail?id=594389) - they've had a bug open since 2016 to provide a proper API for this.
